packages/components/src/components/typography/Code/Code.tsx

```diff
@@ -0,0 +1,20 @@
+import { borders, spacingsPx } from '@trezor/theme';
+import React, { type ReactNode } from 'react';
+import styled from 'styled-components';
+
+const StyledCode = styled.code`
+    font-family: RobotoMono, PixelOperatorMono8, monospace;
+    display: inline;
+    font-size: inherit;
+    line-height: 1.5;
+    font-weight: 400;
+    letter-spacing: -0.4px;
+    padding: 0px ${() => spacingsPx.xxxs};
+    background-color: ${({ theme }) => theme.backgroundNeutralSubtleOnElevation1};
+    box-shadow: inset 0 0 0 1px ${({ theme}) => theme.borderElevation0 };
+    border-radius: ${() => borders.radii.xxs};
```

Comment:
this is not pixel perfect, I would still need more details from the figma, from the image it is very hard to tell. I believe it is close enough for our purpose and could be slightly refactored by someone who has access.